projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"bufio"
+	_ "embed"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+)
+
+//go:embed help.txt
+var helpFile string
+
+func Execute() error {
+	fileNames, numberLine, help := ParsingFlags()
+
+	err := flagHandler(os.Stdout, fileNames, numberLine, help)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func ParsingFlags() (fileNames []string, numberLine *bool, help *bool) {
```

Comment:
I would expect this to return non-pointers, i.e. a `[]string, bool, bool` rather than `[]string, *bool, *bool` - generally returning a pointer happens either because what you're returning is optional (so may be `nil`), or is a value shared with other pieces of code (so multiple references to exactly the same value need to exist).